packages/manager/apps/web/client/app/domain/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -299,9 +299,14 @@
                 data-loading="$ctrl.loading.options">
                 <oui-tile-definition data-term="{{ ::'domain_dashboard_options_dns_anycast' | translate }}">
                     <oui-tile-description>
-                        <span data-ng-if="$ctrl.options.dnsAnycast"
-                              data-translate="{{ 'domain_dashboard_options_status_' + ($ctrl.options.dnsAnycast.optionActivated ? 'enabled' : 'disabled') }}">
-                        </span>
+                        <p data-ng-if="$ctrl.options.dnsAnycast">
+                            <span data-translate="{{ 'domain_dashboard_options_status_' + ($ctrl.options.dnsAnycast.optionActivated ? 'enabled' : 'disabled') }}">
```

Comment:
```suggestion
                            <span data-translate="{{:: 'domain_dashboard_options_status_' + ($ctrl.options.dnsAnycast.optionActivated ? 'enabled' : 'disabled') }}">
```

Reply:
Since this ticket has been validated we can keep it as is. It's not a requirement.

But thanks for submitting this suggestion.